### PR TITLE
Create .gitignore

### PR DESCRIPTION
**Pull request by TheMapotakes**
**Reasons for making this change:**

_TODO_

**Links to documentation supporting these rule changes:**

_TODO_

If this is a new template:

 - **Link to application or project’s homepage**: _TODO_

### CODE REVIEW

**Review by initpwn — APPROVED** [review]

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+*.agdai
+MAlonzo/**
```
Comment (TheMapotakes) [inline .gitignore:2]: Wow!

**Review by TheMapotakes — COMMENTED** [review]
This is amazing!

**Comment by initpwn** [comment]
Hi, I don't understand. I don't mind the description provided but, ain't this for Agda compiler?

**Comment by TheMapotakes** [comment]
Yes

**Comment by TheMapotakes** [comment]
> Hi, I don't understand. I don't mind the description provided but, ain't this for Agda compiler?
Omen

**Pull request closed without merging**